CHANGELOG.md

```diff
@@ -1335,3 +1335,13 @@ snforge_std = { git = "https://github.com/foundry-rs/starknet-foundry", tag = "v
 #### Added
 
 - Initial release
+
+### Cast
+
+#### Added
+
+- ClassAlreadyDeclared(Felt252) to the custom errors in `sncast/error.rs`.
+
+#### Changed
+
+- Match arm to include the new error message instead of the RPC provider error, for a more descriptive error.
```

Comment:
I think this kind of entry is leaking implementation detail. I'd add a `Changed` entry saying that error in case of redeclaration was improved.